src/js/game/LevelMVC/LevelView.js

```diff
@@ -1126,7 +1183,7 @@ module.exports = class LevelView {
         sprite = null;
         if (!levelData.actionPlane[blockIndex].isEmpty) {
           blockType = levelData.actionPlane[blockIndex].blockType;
-          sprite = this.createBlock(this.actionPlane, x, y, blockType);
+          sprite = this.createBlock(this.actionPlane, x, y, blockType, levelData);
```

Comment:
Extra param isn't used?

Reply:
I think it was in one approach I took, but it looks like I ended up going a different route and forgot to remove that. Thanks!